app/controllers/api/v2/images_controller.rb

```diff
@@ -48,6 +48,7 @@ def show
       param_group :image, :as => :create
 
       def create
+        validate_params unless Rails.env.test?
```

Comment:
I believe we should not skip this in the test environment. Please stub the method in tests if you want to skip it.

Reply:
I fixed the problem in a better way, no need for stubs at all now.